pandera/typing/common.py

```diff
@@ -59,6 +59,10 @@
             int,
             str,
             float,
+            List[Any],
+            Dict[Any, Any],
+            Tuple[Any],
+            NamedTuple,
```

Comment:
AFAIK, I can't make the type parameters more precise?

Reply:
did you spot-check this with mypy?

NamedTuple should be fine here

Reply:
Think I've patched the MyPy problems:
```
> pre-commit run mypy --all-files
mypy.....................................................................Passed
- hook id: mypy
- duration: 2.27s

Success: no issues found in 180 source files
```